Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp

```diff
@@ -324,16 +328,15 @@ void URobotVehicleMovementComponent::InitMovementComponent()
     if (bIsFloorHit)
     {
         MinDistanceToFloor = hitResult.Distance;
-        UE_LOG(LogTemp,
-               Warning,
-               TEXT("URobotVehicleMovementComponent::InitMovementComponent - Min Distance To Floor = %f"),
-               MinDistanceToFloor);
     }
+    UE_LOG(LogTemp,

```

Comment:
Does this woring output every tick?

Reply:
Only once at the beginning. This function is called in the constructor.